src/main/java/com/google/step/servlets/CommentGetPostServlet.java

```diff
@@ -33,16 +46,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
       return;
     }
-    List<Comment> comments = manager.getCommentsForDeal(dealId);
+    List<Comment> comments = commentManager.getCommentsForDeal(dealId);
+    List<User> users =
+        comments.stream()
+            .map(comment -> userManager.readUser(comment.userId))
+            .collect(Collectors.toList());
```

Comment:
maybe you can use the readUsers method because according to #57 batch calls are faster, but may need more effort to map users to comments.